admin/data/migrations/migrations/20171215131500-add-pupil-status-code.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const mongoose = require('mongoose')
```

Comment:
@GuyHarwood  - Do we also want to add this migration to the SQL branch?

Reply:
yes.  We discussed this in the friday dev session - this piece of work should be the last with any mongo work, and will require a set of migrations.  You or I can do them with George this week.